ouroboros-consensus/ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Ledger/Forge.hs

```diff
@@ -61,7 +62,7 @@ forgeSimple ForgeExt { forgeExt } cfg forgeState curBlock curSlot tickedLedger t
           simplePrev      = castHash $ getTipHash tickedLedger
         , simpleSlotNo    = curSlot
         , simpleBlockNo   = curBlock
-        , simpleBodyHash  = hash body
+        , simpleBodyHash  = hashWithSerialiser toCBOR body
```

Comment:
Maybe this deserves a helper to, e.g., `hashCBOR`

Reply:
Perhaps, but there was also a deliberate intention to be much more explicit in the way we serialise for hashes :-)

Reply:
Ok, I don't mind.